pasqal_cloud/batch.py

```diff
@@ -63,36 +68,38 @@ class Batch(BaseModel):
     user_id: str
     priority: int
     status: str
-    webhook: Optional[str]
-    _client: Client
+    _client: Client = PrivateAttr(default=None)
     sequence_builder: str
-    start_datetime: Optional[str]
-    end_datetime: Optional[str]
-    device_status: Optional[str]
     ordered_jobs: List[Job] = []
     jobs_count: int = 0
     jobs_count_per_status: Dict[str, int] = {}
+    webhook: Optional[str] = None
```

Comment:
This is the new way of doing it:
https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields